supervisor/supervisor.py

```diff
@@ -102,46 +105,75 @@ def arch(self) -> str:
     async def update_apparmor(self) -> None:
         """Fetch last version and update profile."""
         url = URL_HASSIO_APPARMOR
+
+        # Fetch
         try:
             _LOGGER.info("Fetching AppArmor profile %s", url)
-            async with self.sys_websession.get(url, timeout=10) as request:
+            timeout = aiohttp.ClientTimeout(total=10)
+            async with self.sys_websession.get(url, timeout=timeout) as request:
+                if request.status != 200:
+                    raise SupervisorAppArmorError(
+                        f"Fetching AppArmor Profile from {url} response with {request.status}",
+                        _LOGGER.error,
+                    )
                 data = await request.text()
 
         except (aiohttp.ClientError, asyncio.TimeoutError) as err:
             self.sys_supervisor.connectivity = False
-            _LOGGER.warning("Can't fetch AppArmor profile: %s", err)
-            raise SupervisorError() from err
+            raise SupervisorAppArmorError(
```

Comment:
ClientError and TimeoutError should be separate, TimeoutError does not provide anything for `err`

Reply:
Then I would say, let's ditch to show the exception on the message? Most people don't understand it anyway and we get more details on sentry

Reply:
That works